apps/drec-ui/src/device/containers/card/PublicDeviceCard/PublicDeviceCard.effects.ts

```diff
@@ -12,13 +13,16 @@ export const usePublicDeviceCardEffects = ({
     allDeviceTypes
 }: TUsePublicDeviceCardEffectsArgs) => {
     const navigate = useNavigate();
+    const imageRequestUrl = device.images.length ? device.images : null;
```

Comment:
You don't need this check here. It is already done in image-fetching hook

Reply:
Yeah, the issue was in the entity on the backend. The defaults was '[]'